analyses/molecular-subtyping-MB/02-compare-classes.Rmd

```diff
@@ -249,13 +318,8 @@ print(paste0("Accuracy: ", consensus.corrected.acc))
 consensus.corrected <- out$consensus.mat
 viewDataTable(consensus.corrected)
 
-# merge with clinical ids and save output
-consensus.corrected.out <- clin.merged %>%
-  inner_join(consensus.corrected %>% 
-               mutate(Kids_First_Biospecimen_ID_RNA = Kids_First_Biospecimen_ID) %>%
-               dplyr::select(sample_id, Kids_First_Biospecimen_ID_RNA, molecular_subtype), 
-             by = c('sample_id', 'Kids_First_Biospecimen_ID_RNA'))
-
+# merged output with clinical ids 
+consensus.corrected.out <- out$consensus.mat.out
 write.table(consensus.corrected.out, 
             file = file.path(results_dir, 'MB_batchcorrected_molecular_subtype.tsv'), 
```

Comment:
Have you considered sorting everywhere the TSVs get output by the sample index or similar? I was surprised to see how long the TSV part of the diff was. It would be easier to verify if only the lines that were changed had a diff associated. Sorting won't help this PR, but it will help in the future and is potentially good to implement going forward where smaller TSV files are output.

Reply:
@cgreene oh right, that makes sense. I'll make that change.

Reply:
I also realized that for the first 4 tables in the html,  I have the `Biospecimen` id but for consensus tables I only have the `Participant` ID, so I am going to fix that as well.